server/models/articles.js

```diff
@@ -0,0 +1,64 @@
+const getArticleModel = (sequelize, DataTypes) => {
+  const flags = {
+    timestamps: false,
+    freezeTableName: true
+  };
+  const articleSchema = {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+      unique: true,
+      allowNull: false
+    },
+    title: {
+      type: DataTypes.CHAR,
+      allowNull: false
+    },
+    authorId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      onDelete: "CASCADE"
+    },
+    categoryId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      onDelete: "CASCADE"
+    },
+    body: {
+      type: DataTypes.CHAR,
+      allowNull: false
+    },
+    description: {
+      type: DataTypes.STRING
+    },
+    favourited: {
+      type: DataTypes.BOOLEAN,
+      defaultValue: false,
+      allowNull: false
+    },
+    numberOfReads: {
+      type: DataTypes.INTEGER,
+      defaultValue: 0,
+      allowNull: false
+    },
+    createdAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
```

Comment:
This can be done with a hook, Once again I see no reason timestamps are set to false